scripts/remove

```diff
@@ -100,7 +92,10 @@ fi
 #=================================================
 # REMOVE VARIOUS FILES
 #=================================================
-ynh_script_progression --message="Removing various files..." --time --weight=1
+ynh_script_progression --message="Removing various files..." --weight=1
+
+ynh_secure_remove --file="/usr/bin/dotnet"
+ynh_secure_remove --file="/opt/dotnet"
```

Comment:
will also need to change multi_instance to false then

Reply:
it has been 

https://github.com/YunoHost-Apps/technitium-dns_ynh/blob/a2d9cb93351b22e65a575a9996e2f4d635ae7d80/manifest.json#L25

https://github.com/YunoHost-Apps/technitium-dns_ynh/blob/a2d9cb93351b22e65a575a9996e2f4d635ae7d80/check_process#L16

Reply:
works for me 👍 

Thank you for your effort